README.md

```diff
@@ -31,16 +31,50 @@ npm install --save-dev spectron
 Spectron works with any testing framework but the following example uses
 [mocha](https://mochajs.org):
 
+
+## To use
+To get up and running from your command line:
+```sh
+# Install mocha locally as a dev dependency.
+npm i mocha -D
+# From the root of your project, create a folder called test.
+mkdir test
+# And in that directory, create a file called 'spec.js'
+cd test
+touch spec.js
+```
+
+Then simply include the following your first `spec.js`.
+
 ```js
 var Application = require('spectron').Application
 var assert = require('assert')
+var electronPath = require('electron') // We are actually launching Electron with the binaries included in our node_modules.
+var path = require('path');
 
-describe('application launch', function () {
+describe('Application launch', function () {
   this.timeout(10000)
 
   beforeEach(function () {
     this.app = new Application({
-      path: '/Applications/MyApp.app/Contents/MacOS/MyApp'
+      // Your electron path can be any binary
+      // i.e for OSX an example path could be '/Applications/MyApp.app/Contents/MacOS/MyApp'
+      // But for the sake of the example we fetch it from our node_modules.
+      path: electronPath,
+
+      // Assuming you have the following directory structure
+
+      //  |__ my project
+      //     |__ ...
+      //     |__ main.js
+      //     |__ package.json
+      //     |__ index.html
+      //     |__ ...
+      //     |__ test
+      //        |__ spec.js  <- You are here! ~ Well you should be.
+
+      // The following line tells spectron to look and use the main.js file, and the package.json located 1 level above.
```

Comment:
Do you think it's inconvenient? 

I read the API docs and really, I didn't get it at first glance ~ Had to read it 10 times (honest) even had to go electron definitions to understand how it works. 

Being a visual person, having an image would have helped me alot, and figured it would / could help other people who learn better with images. 

Let me know what you think